ndk-svelte-components/src/lib/utils/notes.ts

```diff
@@ -1,4 +1,4 @@
-import { nip19 } from "nostr-tools";
+import { nprofileEncode, neventEncode, decode } from "nostr-tools/lib/types/nip19";
```

Comment:
must import from `nostr-tools/nip19` instead

Reply:
good catch – this was from before I ended up changing the tsconfig to enable importing in the correct way.